mail-filter/amavisd-new/amavisd-new-2.12.0-r1.ebuild

```diff
@@ -0,0 +1,198 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd
+
+DESCRIPTION="High-performance interface between the MTA and content checkers"
+HOMEPAGE="https://gitlab.com/amavis/amavis"
+SRC_URI="https://gitlab.com/amavis/amavis/-/archive/v${PV}/amavis-v${PV}.tar.bz2"
+
+LICENSE="GPL-2 BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="clamav courier dkim ldap mysql postgres qmail razor rspamd rspamd-https snmp spamassassin zmq"
+
+MY_RSPAMD_DEPEND="( dev-perl/JSON dev-perl/HTTP-Message dev-perl/LWP-UserAgent-Determined )"
+BDEPEND="acct-user/amavis"
+RDEPEND="${BDEPEND}
+	>=dev-lang/perl-5.10.0
+	app-arch/arc
+	app-arch/bzip2
+	app-arch/cabextract
+	app-arch/cpio
+	app-arch/gzip
+	app-arch/lha
+	app-arch/lrzip
+	app-arch/lzop
+	app-arch/ncompress
+	app-arch/p7zip
+	app-arch/pax
+	app-arch/unarj
+	app-arch/unrar
+	app-arch/xz-utils
+	app-arch/zoo
+	clamav? ( app-antivirus/clamav )
```

Comment:
In proxy-maint we sort this list by separating all conditional and unconditional deps.